api-server/src/server/boot/challenge.js

```diff
@@ -223,14 +230,19 @@ export function modernChallengeCompleted(req, res, next) {
     .getCompletedChallenges$()
     .flatMap(() => {
       const completedDate = Date.now();
-      const { id, files } = req.body;
+      const { id, files, challengeType = null } = req.body;
```

Comment:
```suggestion
      const { id, files, challengeType } = req.body;
```
Nitpick: `undefined` is fine here.

Reply:
fixed